Tarea/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
The name of your folder should be `firstname_lastname`

Reply:
This item was changed.